src/main/kotlin/racingcar/random/RandomGeneratorImpl.kt

```diff
@@ -0,0 +1,9 @@
+package racingcar.random
+
+import kotlin.random.Random
+
+class RandomGeneratorImpl : RandomGenerator {
+    override fun generate(): Int {
+        return Random.nextInt(0, 10)
```

Comment:
소소하지만 이런식으로 표현할수도 있습니다 ㅎㅎ
```suggestion
override fun generate(): Int = (0..9).random()
```

Reply:
요거는 한줄짜리라 고민하다가.. 그냥 전체적으로 함수는 expression body function 을 사용하지 말자라고 하고있었습니다!

혹시 이렇게 = 으로 선언하는 함수를 사용하시는 기준이 있으실까요?